### ISSUE TITLE
question on load_files() function

### ISSUE THREAD

**Issue by jermaine1ronquillo** [NeerajSarwan/CPT#1]
looking at the code for the load_files() function, why did you append the values of the test file in the **data** list?

`for index, row in test.iterrows():
    data.append(row.values)
    target.append(list(row.values))`

**Comment by NeerajSarwan** [comment]
Given, the sequential nature of the problem, test set can be merged with the train set to create a bigger training dataset. Hence the append.

**Comment by jermaine1ronquillo** [comment]
Thank you for the response.

Can the merging be optional? And I'm looking forward for the python implementation of the other algorithms (when can this be?). And one last thing, if you have any idea on time series that can be made into an example using CPT is a great help.

Again thank you

**Comment by NeerajSarwan** [comment]
Yes, it can be optional and I think it should make more sense too. I will update the code with the changes. keep an eye out.

**Comment by Sandy4321** [comment]
Given, the sequential nature of the problem, test set can be merged with the train set to create a bigger training dataset. Hence the append.
it is mistaken to add test data for training, they call it data leaking. can you test your code without such a bug?

**Comment by NeerajSarwan** [comment]
@Sandy4321 I am well aware about what data leaking is. But this isn't the case of data leaking as far as I know. 
Also, there merging of train and test is not mandatory as I have mentioned above. Feel free to fork the repository and test out the code.

**Comment by NeerajSarwan** [comment]
@jermaine1ronquillo, I have removed the inconsistency. The train and test append operation should be consistent now. Thanks for pointing it out.
